docs/general/wallets.md

```diff
@@ -0,0 +1,155 @@
+polkadot js plus which completely re-developed, with a new look, smooth user experience, and many
+performance improvements. Supports Proxy accounts and Social recovery. Address (Watch) Only Accounts, 
+Parachain Accounts, Custom Remote Nodes, Phishing Detection, QR Account Import, Export/Import/Derive 
+account(s). 
+
+### SubWallet
+
+Web3 Multiverse Gateway for Polkadot & Kusama Ecosystem. Track, send, receive and monitor
+multi-chain assets with ease. XCM Transfer, NFT Management, Parity Signer & Ledger Support, Light
+Clients Support, QR-code Account Import, Token Import, EVM DApp Support, MetaMask Compatibility,
+Custom Endpoint, Fiat On-ramp. Subwallet received funding from Polkadot Treasury
+[[1](https://polkadot.polkassembly.io/treasury/138),
+[2](https://polkadot.polkassembly.io/treasury/162)].
+
+### Talisman
+
+A better way to explore Web3. Keep your assets safe, manage your portfolio and explore Polkadot and
+Ethereum apps with Talisman. Interact with Web3 apps, store your favourite crypto assets and manage
+your accounts on over 150+ Substrate and EVM networks. NFT Management, Ledger Support, Fiat On-ramp,
+Portfolio tracking. Talisman received funding from Polkadot Treasury
+[[1](https://polkadot.polkassembly.io/treasury/148)].
+
+### Fearless Wallet
+
+The DeFi Wallet for the Future Android and iOS native mobile Polkadot/Kusama wallet for the Dotsama
+ecosystem with emphasis on UX, performance, and security. Get access to charts to better understand
+your profits and how to maximize your staking performance. The goal of Fearless Wallet is to
+radically expand access to decentralized finance (DeFi) by making complex functions much easier to
+use and understand. Fearless Wallet received funding from Kusama Treasury
+[[1](https://kusama.polkassembly.io/treasury/23), [2](https://kusama.polkassembly.io/treasury/34),
+[3](https://kusama.polkassembly.io/treasury/74), [4](https://kusama.polkassembly.io/treasury/102),
+[5](https://kusama.polkassembly.io/treasury/178)].
+
+### Nova Wallet
+
+OpenGov, Governance v1, XCM Transfers, Parity Signer & Ledger Support, DApp Support with Polkadot JS
+and Metamask/EVM Integration. Nova Wallet received funding from Kusama Treasury
+[[1](https://kusama.polkassembly.io/treasury/122),
+[2](https://kusama.polkassembly.io/treasury/158)].
+
+### Polkawallet
+
+Polkawallet provides cross-chain asset one-stop management, convenient staking, governance, and
+multiple DeFi services, the private key is self-owned. Polkawallet received funding from Kusama
+Treasury [[1](https://kusama.polkassembly.io/treasury/32),
+[2](https://kusama.polkassembly.io/treasury/41)].
+
+## Supported Wallets
+
+[Polkadot support](https://support.polkadot.network/) can provide assistance with issues related to
+Polkadot-JS UI, the Polkadot-JS extension, or Parity Signer. For other wallet software, you should
+contact the developers of that wallet. Please note that these wallets are quite bare bones and
```

Comment:
I would get rid of "bare bones", replace with "oriented towards developers and power users."